packages/slate-react/src/components/editable.tsx

```diff
@@ -110,6 +111,7 @@ export const Editable = (props: EditableProps) => {
     readOnly = false,
     renderElement,
     renderLeaf,
+    scrollSelectionIntoView,
```

Comment:
@ulion thank you for this! Love this idea of making it customizable.

Can we make this prop have a default which implements the existing logic, ie:

```ts
scrollToSelection = defaultScrollToSelection,
```

Which is defined as:

```ts
const defaultScrollToSelection = (editor) => {
  const { selection } = editor
  const domRange = ReactEditor.toDOMRange(editor, selection)
  const leafEl = newDomRange.startContainer.parentElement!
  scrollIntoView(leafEl, {
    scrollMode: 'if-needed',
    boundary: el,
  })
}
```